### PR TITLE
[3.1.x][5610] Update the Pull from Remote dialog

### PR DESCRIPTION
**Pull request by jvega190**
- https://github.com/craftercms/craftercms/issues/5610 - revert changes
- https://github.com/craftercms/craftercms/issues/5615
- https://github.com/craftercms/craftercms/issues/5618

### CODE REVIEW

File: ui/app/src/components/BlueprintForm.tsx [hunk 1]
```diff
@@ -135,6 +135,9 @@ function BlueprintForm(props: BlueprintFormProps) {
       setInputs({ blueprintFields: parameters });
     } else {
       setInputs({ [e.target.name]: e.target.value });
+      if (blueprint.id === 'GIT' && e.target.name === 'repoRemoteBranch') {
+        setInputs({ sandboxBranch: e.target.value });
+      }
```
Comment (rart) [inline ui/app/src/components/BlueprintForm.tsx]:
```suggestion
      setInputs({ [e.target.name]: e.target.value, ...(blueprint.id === 'GIT' && e.target.name === 'repoRemoteBranch' && { sandboxBranch: e.target.value }) });
```

**Review by rart — COMMENTED** [review]
@jvega190 please compare the rollbacks with the original PR and re-confirm the changes are all correct.

**Comment by jvega190** [comment]
Checked the rollback, all good

**Comment by jvega190** [comment]
@rart I pushed the fix for https://github.com/craftercms/craftercms/issues/5618 in here to avoid merge conflicts.

**Review by rart — APPROVED** [review]

**Pull request merged**